test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateScripts.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.DotNet.ProjectJsonMigration.Tests
 {
     public class GivenThatIWantToMigrateScripts : TestBase
     {
+        private const bool IsMultiTFM = true;
```

Comment:
Seems we should have a default value. Having to pass this in every test distracts from what is being tested.